tests/integration/cmor/_fixes/cmip5/test_bcc_csm1_1_m.py

```diff
@@ -1,32 +1,28 @@
 """Test fixes for bcc-csm1-1-m."""
-import unittest
 
 from esmvalcore.cmor._fixes.cmip5.bcc_csm1_1_m import Cl, Tos
 from esmvalcore.cmor._fixes.common import (
     ClFixHybridPressureCoord,
     OceanFixGrid,
 )
-from esmvalcore.cmor._fixes.fix import Fix
+from esmvalcore.cmor._fixes.fix import AutomaticFix, Fix
 
 
 def test_get_cl_fix():
     """Test getting of fix."""
     fix = Fix.get_fixes('CMIP5', 'bcc-csm1-1-m', 'Amon', 'cl')
-    assert fix == [Cl(None)]
+    assert fix == [Cl(None), AutomaticFix(None)]
```

Comment:
Maybe you could write this type of test like this
```python
    assert Cl(None) in Fix.get_fixes('CMIP5', 'bcc-csm1-1-m', 'Amon', 'cl')
```
to avoid having to import AutomaticFix everywhere (and us having to change all of these tests again next time there is an update. If there are a few tests that check the complete list of tests that is returned, that should be enough, right?

Reply:
I actually like that we test that ALL fixes are applied to each model. We have similiar tests for datasets that have an `AllVars` fix. In addition, I spent at least an hour to make these changes yesterday, an I really don't want to go trough 100 files again.